stable/redis-ha/README.md

```diff
@@ -70,6 +70,7 @@ The following tables lists the configurable parameters of the Redis chart and th
 | `tolerations`                    | Toleration labels for pod assignment                  | []                                                        |
 | `servers.serviceType`            | Set to "LoadBalancer" to enable access from the VPC   | ClusterIP                                                 |
 | `servers.annotations`            | See Appliance mode                                    | ``                                                        |
+| `rbac.create`                    | create rbac resources                                 | false                                                     |
```

Comment:
I would personally call this rbac.enabled instead.
If rac.enabled {
...
}

Reply:
hm, i checked the other charts in the repo and most of them seem to use the rbac.create flag. So for consistency i would prefer to keep the rbac.create.

Reply:
@RoelVanNyen looks like @smelchior is right on this -> https://github.com/kubernetes/charts/search?q=%22rbac.create%22&type=Code&utf8=%E2%9C%93

Can this PR go ahead?